### DISCUSSION TITLE
How can I hide an element on click?

### DISCUSSION THREAD

**Discussion by datajoely** [slidevjs/slidev#214] [Q&A] [answered]
I want to make an element transition out on a certain click. This is the reverse of the 'click in workflow' and would be called an exit animation in PPT lingo.

Thanks! Loving using slidev so far!

**Comment by antfu** [comment, marked as answer]
We have `v-click-hide` directive, which I forget to document it. Thanks for the reminder

**Reply by datajoely** [reply]
Amazing!
